### DISCUSSION TITLE
Strapi - Overlapping layout

### DISCUSSION THREAD

**Discussion by smsalmanshahAN** [an-nadaaa/nda-front#126] [Feedback]
![image](https://user-images.githubusercontent.com/101787508/161207894-fcf0a063-4135-493d-8a27-122cd89d4401.png)

The closed and featured buttons are overlapping. It is workable but needs to be fixed at the end (after high priority issues and features are completed)

**Comment by an-nadaa** [comment]
@smsalmanshahAN This is unfortunately not something we want to fix as it's outside of our scope. however you can raise an issue [here](https://github.com/strapi/strapi/issues) if you believe this is really needed.
